src/EPR.SubsidiaryBulkUpload.Application/Services/SubsidiaryService.cs

```diff
@@ -104,6 +105,32 @@ public async Task<bool> GetSubsidiaryRelationshipAsync(int parentOrganisationId,
         return orgResponse.FirstOrDefault();
     }
 
+    public async Task<OrganisationResponseModel?> GetCompanyByLocalDBCompanyName(string companiesHouseName)
+    {
+        var response = await _httpClient.GetAsync($"{OrganisationByCompanyHouseNameUri}?companiesHouseName={companiesHouseName}");
```

Comment:
You should encode companiesHouseName when using it in a url - what happens if the company is called "Bob & Alice's" for example

Reply:
done